yocaml_markdown.opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+version: "dev"
+synopsis: "YOCaml Markdown processor"
+maintainer: "xhtmlboi@gmail.com"
+authors: [
+  "The XHTMLBoy <xhtmlboi@gmail.com>"
+  "Pierre Ruyter <grimfw@gmail.com>"
+]
+
+build: [
+  [ "dune" "subst" ]
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name ] {with-test}
+  [ "dune" "build" "@doc" "-p" name ] {with-doc}
+]
+
+license: "GPL-3.0-or-later"
+tags: [ "shell" "bin" "make" "static" "blog" "generator" ]
+homepage: "https://github.com/xhtmlboi/yocaml"
+dev-repo: "git://github.com/xhtmlboi/yocaml.git"
+bug-reports: "https://github.com/xhtmlboi/yocaml/issues"
+
+depends: [
+  "ocaml" { >= "4.11.1" }
+  "dune" { >= "2.8" }
+  "odoc" {with-doc}
+  "yocaml"
+  "omd" { >= "1.3.1" }
+]
+
+pin-depends: [
+  ["preface.dev" "git+ssh://git@github.com/xvw/preface.git"]
+]
```

Comment:
I suggest to use `git://github.com/xvw/preface.git` (HTTPS) instead of (SSH) for CI purpose. And you should add `git://github.com/xhtmlboi/yocaml.git` (and `{pinned}` decorator on `yocaml`).